shell/platform/android/io/flutter/embedding/engine/mutatorsstack/FlutterMutatorsStack.java

```diff
@@ -60,6 +61,19 @@ public FlutterMutator(Rect rect) {
       this.rect = rect;
     }
 
+    /**
+     * Initialize a clip rrect mutator.
+     *
+     * @param rect the rect of the rrect
+     * @param radiis the radiis of the rrect. Array of 8 values, 4 pairs of [X,Y]. This value cannot
+     *     be null.
+     */
+    public FlutterMutator(Rect rect, float[] radiis) {
```

Comment:
We should refactor this class such as FlutterMutator is an interface, and there are individual classes for each type of mutation. The issue is that having a single type that serves the purposes of multiple types is harder to maintain.

Reply:
this can be done later though.